git/git.go

```diff
@@ -490,13 +490,45 @@ func (c *gitConfig) IsGitVersionAtLeast(ver string) bool {
 	return IsVersionAtLeast(gitver, ver)
 }
 
+// Prepend Git config instructions to disable Git LFS filter
+func PrependConfigToDisableLFS(args ...string) []string {
```

Comment:
> re 2) My thinking was that "disablingLFS" should be the "default" case, no?

I would prefer that this not be the default case. I think that this approach will make it clearer that by calling `exec()`/`simpleExec()` you have to explicitly opt-in to not using LFS.

I agree though that this approach makes it harder for new callers to know when to add this and when not to. What do you think about `execWithLFS` and `execWithoutLfs` as another alternative?